Source/WTF/wtf/text/StringView.cpp

```diff
@@ -259,7 +259,7 @@ static AtomString convertASCIILowercaseAtom(const CharacterType* input, unsigned
             StringImpl::copyCharacters(characters, input, i);
             for (; i < length; ++i)
                 characters[i] = toASCIILower(input[i]);
-            return result;
+            return AtomString(result);
```

Comment:
We need a better idiom for this. Like the code above in `Coder<AtomString>::decode`, for a short string it would be way better to not allocate memory for a copy of the characters and instead have a way to go straight into the atom string table. I guess the saving grace here is that it’s only when a character is uppercase.